packages/components/src/components/FiltersForm.vue

```diff
@@ -1,19 +1,26 @@
 <template>
   <form :class="classes" @submit.prevent="onFormSubmit">
     <input
+      ref="input"
       class="filters__form-input"
+      event
```

Comment:
What is this attribute?

Reply:
That was a typo, fixed https://github.com/applandinc/appmap-js/pull/492/commits/2d704c28ff38563ef65507372ca09404ad586b9a